test/Farmer/addLocation.test.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import nock from "nock";
+import { describe, it } from "mocha";
+import {
+  counties,
+  regions,
+  locationError,
+  geoArea,
+} from "./farmerResponses.js";
+import { BASEURL } from "../../src/core/urls.js";
+import {
+  getLocations,
+  getRegions,
+} from "../../src/users/farmer/listlocations.js";
+// import { addLocation } from '../../src/core/usermanagement.js';
+
+describe("Locality details", () => {
+  beforeEach(() => {
+    nock(`${BASEURL}`).get("/ussd/regions/").reply(200, regions);
+    nock(`${BASEURL}`).get("/ussd/counties/1").reply(200, counties);
+    nock(`${BASEURL}`).get("/ussd/counties/99").reply(404, locationError);
+    nock(`${BASEURL}`).post("/ussd/geoarea/").reply(201, geoArea);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
    nock(`${BASEURL}`).post('/ussd/geoarea/').reply(201, geoArea);
```